contracts/truefi/TrueLender.sol

```diff
@@ -240,6 +253,16 @@ contract TrueLender is ITrueLender, Ownable {
         emit RiskAversionChanged(newRiskAversion);
     }
 
+    /**
+     * @dev Set new loans limit. Only owner can change parameters.
+     * @param newLoansLimit New loans limit
+     */
+    function setLoansLimit(uint256 newLoansLimit) external onlyOwner {
+        require(newLoansLimit >= _loans.length, "TrueLender: Cannot set loans amount to less than active loans");
```

Comment:
Not sure about this require - in theory we might want to change it to a value lower and don't accept any loans till old ones expire

Reply:
Not sure if it will ever happen, but it doesn't hurt us to leave this option

Reply:
Do You mean setter or require. Ive got a little confused there

Reply:
require